agent/config/config.go

```diff
@@ -58,6 +58,14 @@ func Parse(data string, format string) (c Config, md mapstructure.Metadata, err
 	return c, md, nil
 }
 
+// Cache is the tunning configuration for cache, values are optional
+type Cache struct {
+	// EntryFetchMaxBurst max burst size of RateLimit for a single cache entry
+	EntryFetchMaxBurst *int `json:"entry_fetch_max_burst,omitempty" hcl:"entry_fetch_max_burst" mapstructure:"entry_fetch_max_burst"`
+	// EntryFetchRateLimit represents the max calls/sec for a single cache entry
+	EntryFetchRateLimit *float64 `json:"entry_fetch_rate_limit,omitempty" hcl:"entry_fetch_rate_limit" mapstructure:"entry_fetch_rate_limit"`
```

Comment:
The rpc equivalent is called `rpc_rate`, what do you think about using `entry_fetch_rate` here?

Reply:
Done in b3947c3f218b1e616c05d8f4cb0876cf7a543cd6